webapp/javascript/components/TimelineChart/TimelineChartSelection.ts

```diff
@@ -116,8 +117,14 @@ const handleHeight = 22;
       return isLeftSelecting ? 'left' : isRightSelecting ? 'right' : null;
     }
 
-    function setCursor(type: string) {
-      $('canvas.flot-overlay').css('cursor', type);
+    function cursor() {
+      let current = '';
+      return function (type: string) {
+        if (type && current !== type) {
```

Comment:
Why do you have to check `type` is truthy? If it's always truthy we don't need to check it, and if it's not truthy then we need to indicate that in the types (ie `(type: string)` becomes `(type?: string)`)